var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -179,6 +183,13 @@ class Seacas(CMakePackage):
         when="@:2021-01-20",
         msg="The Faodel TPL is only compatible with @2021-04-05 and later.",
     )
+    # Remove use of variable in array assignment (triggers c2057 on MSVC)
```

Comment:
I have fixed this issue in the current version of SEACAS, so the patch should no longer be needed once I put up the next version.

Reply:
I version scoped the patch, it will only be applied to versions equal to the current version in Spack or older.